.gitignore

```diff
@@ -331,3 +331,5 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
+appSettings.json
```

Comment:
Why do we want to ignore this? I normally see it included in checkins.

Reply:
Removed it. I thought it might be better to include it to prevent accidentally checking in connection strings.